src/neptune_scale/sync/sync_process.py

```diff
@@ -414,13 +289,12 @@ def submit(self, *, operation: RunOperation) -> Optional[SubmitResponse]:
 
     def work(self) -> None:
         try:
-            while (operation := self.get_next()) is not None:
-                sequence_id, timestamp, data = operation
+            generator = _stream_operations(self._operations_repository, self._metadata.run_id, self._metadata.project)
+            while (operation := next(generator, None)) is not None:
+                run_operation, sequence_id, timestamp = operation
 
                 try:
-                    logger.debug("Submitting operation #%d with size of %d bytes", sequence_id, len(data))
-                    run_operation = RunOperation()
-                    run_operation.ParseFromString(data)
+                    logger.debug("Submitting operation #%d with size of %d bytes", sequence_id, len(""))
```

Comment:
Either remove or fill with something sensible (ByteSize is probably too heavy, but maybe snapshot count / create run count, or something?)

Reply:
I will add if for debug level: 
```
if logger.isEnabledFor(logging.DEBUG):
   logger.debug(
                            "Submitting operation #%d with size of %d bytes", sequence_id, run_operation.ByteSize()
                        )
```